javascript/ql/src/experimental/Security/CWE-347-noVerification/JsonWebTokenNotWorking.ql

```diff
@@ -0,0 +1,66 @@
+/**
+ * @name This query is for seeing if we can have two taint config within on query file
```

Comment:
This does not look like a file you meant to include? 
This look like code you used for testing.

Reply:
Oh, I think I know why.  
The `DataFlow::PathNode` includes which `DataFlow::Configuration` it came from. 
So you need to check that the `DataFlow::Node` that you get from calling `.getNode()` on the source is the same between the two "sources", you can't use the `DataFlow::PathNode` from one configuration directly in another.